contracts/near/eth-prover/src/lib.rs

```diff
@@ -4,9 +4,6 @@ use eth_types::*;
 use near_sdk::{env, ext_contract, near_bindgen, Gas, PanicOnDefault, PromiseOrValue};
 use rlp::Rlp;
 
-#[cfg(test)]
-mod tests;
-
```

Comment:
Now there is a file `tests.rs` which is unused.

Reply:
Ohh, I see, thanks for pointing that. Will move the test over there.